ballerina/Module.md

```diff
@@ -1,7 +1,32 @@
+
+This module supports [Salesforce v59.0 REST API](https://developer.salesforce.com/docs/atlas.en-us.224.0.api_rest.meta/api_rest/intro_what_is_rest_api.htm), [Salesforce v59.0 APEX REST API](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_rest_intro.htm), [Salesforce v59.0 BULK API](https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/api_asynch_introduction_bulk_api.htm), and [Salesforce v59.0 BULK V2 API](https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/bulk_api_2_0.htm).
+
+## Setup guide
+
+1. Create a Salesforce account with the REST capability.
+2. Go to Setup --> Apps --> App Manager 
+  ![Setup Side Panel](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-sfdc/revamp-2023/docs/side-panel.png)
+3. Create a New Connected App
+![Create Connected Apps](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-sfdc/revamp-2023/docs/create-connected-apps.png)
+  - Here we will be using https://test.salesforce.com as we are using sandbox enviorenment. Users can use https://login.salesforce.com for normal usage
+  ![Create Connected Apps](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-sfdc/revamp-2023/docs/create_connected%20_app.png)
+4. After the creation user can get consumer key and secret through clicking on the `Manage Consume Details` button.
+![Consumer Secrets](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-sfdc/revamp-2023/docs/crdentials.png)
+5. Next step would be to get the token.
+  - Log in to salesforce in your prefered browser and enter the following url 
+  `https://<YOUR_INSTANCE>.salesforce.com/services/oauth2/authorize?response_type=code&client_id=<CONSUMER_KEY>&redirect_uri=<REDIRECT_URL>`
+  - Allow access if an alert pops up and the browser will be redirected to a Url like follows.
+  `https://login.salesforce.com/?code=<ENCODED_CODE>`
+  - the code can be obtained after decoding the encoded code
+
+6. Get Access and Refresh tokens
+ - following request can be sent to obtain the tokens
+ ```curl -X POST https://<YOUR_INSTANCE>.salesforce.com/services/oauth2/token?code=<CODE>&grant_type=authorization_code&client_id=<CONSUMER_KEY>&client_secret=<CONSUMER_SECRET>&redirect_uri=https://test.salesforce.com/``` 
+ - tokens can be obtaind from the response
+
 
-This module supports [Salesforce v48.0 REST API](https://developer.salesforce.com/docs/atlas.en-us.224.0.api_rest.meta/api_rest/intro_what_is_rest_api.htm).
 
 ## Prerequisites
```

Comment:
Lets follow the format on the newly released modules, twilio, calendar

Reply:
updated in [6df510d](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/311/commits/6df510d1098d680d7b11812ca4ebd7b34b2af897)